client/src/ui/base/_themes.scss

```diff
@@ -5,7 +5,6 @@
 @mixin light-theme {
   --text-primary: #{$mdn-theme-light-text-primary};
   --text-secondary: #{$mdn-theme-light-text-secondary};
-  --text-active: #{$mdn-theme-light-text-active};
```

Comment:
Can we remove the $mdn-theme-light-text-active definition as well then?

Reply:
Fixed in 6415c73942.